pf2/predict.py

```diff
@@ -76,35 +84,44 @@ def predict_mortality(data: pd.DataFrame, meta: pd.DataFrame, proba: bool = Fals
             labels (pd.Series): classification targets
         else:
             accuracy (float): prediction accuracy
-            coefficients (pd.DataFrame): LR model coefficients; columns
-                correspond to unique models for COVID/non-COVID patients
+            models (tuple[COVID, Non-COVID]): fitted PLSR models
     """
     if not isinstance(data, pd.DataFrame):
         data = pd.DataFrame(data)
 
-    data = data.loc[meta.loc[:, "patient_category"] != "Non-Pneumonia Control", :]
-    meta = meta.loc[meta.loc[:, "patient_category"] != "Non-Pneumonia Control", :]
+    data = data.loc[
+        meta.loc[:, "patient_category"] != "Non-Pneumonia Control",
```

Comment:
@JacksonLChin Does this remove patients who fall under the non-pneumonia control?

Reply:
Yup, but just for the mortality predictions! I think it makes sense to remove these patient since we're primarily interested in pneumonia-related mortality.